.github/workflows/rolling-instance-actions.yml

```diff
@@ -0,0 +1,74 @@
+# actions workflow for PRs
+name: rolling-instance-actions
+
+on:
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  actions:
+    runs-on: windows-latest
+    env:
+      RUNNER_DEBUG: 1
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        lfs: true
+
+    - name: Test who-am-i action
+      uses: ./who-am-i
```

Comment:
just to get some coverage on the actions calling into pac and SoPa: I recommend we keep these 2 actions in the PR workflow